traffic_mirroring.go

```diff
@@ -30,36 +25,36 @@ func main() {
 	}
 
 	packetSource := gopacket.NewPacketSource(handle, handle.LinkType())
-	max_size := 10
-	packets_list := make([][]byte,max_size) 
-	current_packet := 0
+	packetsList := make(chan []byte, 22)
+	filePath := "Test"
```

Comment:
Please, give a more suggestive filename. This code might be executed in multiple instances.